### ISSUE TITLE
Activated Joomla Cache overwrites custom badge color in the Article Type Options

### ISSUE THREAD

**Issue by WM-Loose** [templaza/astroid-framework#308]
Joomla 4.2.9 PHP8.1
Joomlaplates Template 1.3.12
Astroid Framework 2.6.4

Problem:
Activated Joomla Cache overwrites custom badge color in the Article Type Options with the primary color specified in the template. Adjusting the post and saving will display the custom color but as soon as the webpage refreshes, the primary color is back. If I deactivate the Joomla cache (normal cache in file), it is also displayed after the refresh as specified in the contribution.

![1](https://user-images.githubusercontent.com/99647423/227783139-96596705-bb81-42f9-8897-95c0000a361d.png)

![2](https://user-images.githubusercontent.com/99647423/227783141-b0bb4cb1-0dcd-4980-abc8-064b41d56c7e.png)

after save in the article:
![3](https://user-images.githubusercontent.com/99647423/227783463-c7343991-659b-4e71-98dd-f94db374bbe4.png)

after refresh site frontend (take the red color from template options primary:
![4](https://user-images.githubusercontent.com/99647423/227783497-2cf6d29c-07ca-4c52-91da-933ad0da2329.png)

**Comment by WM-Loose** [comment]
New Test! It´s not a astroid problem. I close the call now.

**Comment by Joomlaplates** [comment]
Yes I confirm, when Joomla cache is enabled the custom badge jumps back to primary color.

**Comment by felixkat** [comment]
Closing due to age of issue.  I also tested with Joomla 5 and Astroid 3.1.12 and it works as expected.
